src/pint/utils.py

```diff
@@ -1138,18 +1179,56 @@ def pulsar_age(f, fdot, n=3, fo=1e99 * u.Hz):
 
     Return the age of a pulsar given the spin frequency
     and frequency derivative.  By default, the characteristic age
-    is returned (assuming a braking index 'n'=3 and an initial
-    spin frequency fo >> f).  But 'n' and 'fo' can be set.
+    is returned (assuming a braking index `n` =3 and an initial
+    spin frequency `fo` >> `f`).  But `n` and `fo` can be set.
+
+    Parameters
+    ----------
+    f : :class:`astropy.units.Quantity`
+        pulsar frequency
+    fdot : :class:`astropy.units.Quantity`
+        frequency derivative
+    n : int, optional
+        braking index (default = 3)
+    fo : :class:`astropy.units.Quantity`, optional
+        initial frequency
+
+    Returns
+    -------
+    age : :class:`astropy.units.Quantity`
+        pulsar age in ``u.yr``
+
+    Notes
+    -----
+    Calculates ``(-f/(n-1)/fdot) * (1 - (f/fo)**(n-1))``
```

Comment:
Our Sphinx supports LaTeX math, maybe that would be clearer?

Reply:
Sure. Do you have any examples you could point to, in order to get the backslashes etc correct?

Reply:
Er. I believe, in python, if you make the docstring a "raw string" (start with r""" ... """) the backslashes should be no different from those in normal LaTeX. But I could be wrong.